packages/base/src/formbuilder/editform.tsx

```diff
@@ -117,7 +117,6 @@ export class EditForm extends React.Component<IEditFormProps, any> {
                 this.syncObjectProperties(this.props.source, properties);
               }}
               formChangedSignal={this.sourceFormChangedSignal}
-              sourceType={source?.type || 'RasterSource'}
```

Comment:
See other comment about `ISourceFormProps`, we should properly type `SourceForm` here and have it take sourceType as input.